storage/innobase/xtrabackup/test/inc/keyring_kmip.sh

```diff
@@ -0,0 +1,59 @@
+#
+# keyring_kmip helpers
+# Read KEYRING_TYPE[component|plugin] to decide which keyring type to setup
+# If KEYRING_TYPE is not set, we assume plugin
+
+. inc/common.sh
+
+if [ -z ${KEYRING_TYPE+x} ]; then
+  KEYRING_TYPE="plugin"
+fi
+
+if test -d $PWD/../../../../plugin_output_directory
+then
+  plugin_dir=$PWD/../../../../plugin_output_directory
+else
+  plugin_dir=$PWD/../../lib/plugin/
+fi
+XB_EXTRA_MY_CNF_OPTS="${XB_EXTRA_MY_CNF_OPTS:-""}
+xtrabackup-plugin-dir=${plugin_dir}
+"
+
+#placeholder
+keyring_args=""
+
+KMIP_SERVER_ADDR="${KMIP_SERVER_ADDR:-127.0.0.1}"
+KMIP_SERVER_PORT="${KMIP_SERVER_PORT:-5696}"
+KMIP_CLIENT_CA="${KMIP_CLIENT_CA:-/etc/pykmip/client_certificate_jane_doe.pem}"
+KMIP_CLIENT_KEY="${KMIP_CLIENT_KEY:-/etc/pykmip/client_key_jane_doe.pem}"
+KMIP_SERVER_CA="${KMIP_SERVER_CA:-/etc/pykmip/root_certificate.pem}"
```

Comment:
how to get these files? can you please add a comment above?

Reply:
Done